ls1intum/Artemis
# [Exam] Client: Adjust summary exercise card of programming exercises

### Checklist

### Motivation and Context


Fixes the faulty repository link and changes the summary card of programming exercises
### Description

- removed problem statement in summary card of programming exercises
- added repository link
### Steps for Testing


1. Create an exam with programming exercise(s)
1.1 add yourself (or any student) to the exam
1.2 make sure to click generate individual exams + prepare exercise start
2. Participate in that exam
3. Wait until exam is over
4. Summary Page should appear
4.1 Check whether your submission, the title and link are correct

### Screenshots


![image](https://user-images.githubusercontent.com/50466283/86042729-e07db300-ba47-11ea-84cf-7071265240d4.png)
